src/server/routes/columns.ts

```diff
@@ -54,6 +55,26 @@ export default async (fastify: FastifyInstance) => {
     return data
   })
 
+  fastify.get<{
```

Comment:
I've opted to expose these separately rather than have the existing endpoints handle both cases; it gets annoying with the GET accepting a single ID param (and just gnarly in any case).

We can replace the current endpoint as part of a breaking change in the future.